tests/otel_integrations/test_httpx.py

```diff
@@ -240,20 +240,20 @@ async def test_async_httpx_client_instrumentation_with_capture_headers(
     assert all(key in span['attributes'] for key in expected_attributes)
 
 
-CAPTURE_JSON_BODY_PARAMETERS: tuple[tuple[str, ...], list[tuple[str, Any, dict[str, Any]]]] = (
+CAPTURE_JSON_BODY_PARAMETERS: tuple[tuple[str, ...], list[Any]] = (
     ('content_type', 'body', 'expected_attributes'),
     [
-        ('application/json', '{"hello": "world"}', {'http.request.body.json': '{"hello": "world"}'}),
-        ('application/json; charset=utf-8', '{"hello": "world"}', {'http.request.body.json': '{"hello": "world"}'}),
+        ('application/json', '{"hello": "world"}', {'http.request.body.text': '{"hello": "world"}'}),
+        ('application/json; charset=utf-8', '{"hello": "world"}', {'http.request.body.text': '{"hello": "world"}'}),
         (
             'application/json; charset=iso-8859-1',
             '{"hello": "world"}',
-            {'http.request.body.json': '{"hello": "world"}'},
+            {'http.request.body.text': '{"hello": "world"}'},
         ),
-        ('application/json; charset=utf-32', '{"hello": "world"}', {'http.request.body.json': '{"hello": "world"}'}),
-        ('application/json; charset=potato', '{"hello": "world"}', {'http.request.body.json': '{"hello": "world"}'}),
-        ('application/json; charset=ascii', b'\x80\x81\x82', {'http.request.body.json': '���'}),
-        ('application/json; charset=utf8', b'\x80\x81\x82', {'http.request.body.json': '���'}),
+        ('application/json; charset=utf-32', '{"hello": "world"}', {}),
+        ('application/json; charset=potato', '{"hello": "world"}', {}),
+        ('application/json; charset=ascii', b'\x80\x81\x82', {}),
+        ('application/json; charset=utf8', b'\x80\x81\x82', {}),
         ('text/plain', '{"hello": "world"}', {}),
         ('', '{"hello": "world"}', {}),
```

Comment:
Why is this not capturing? The content type shouldn't matter any more

Reply:
I've changed a bit the test to make sure the field is there.